etc/spgw_u.conf

```diff
@@ -118,7 +118,8 @@ SPGW-U =
        ENABLE_5G_FEATURES = "@ENABLE_5G_FEATURES@" # Set to 'yes' to support 5G Features
        REGISTER_NRF = "@REGISTER_NRF@";  # Set to 'yes' if UPF resgisters to an NRF
        USE_FQDN_NRF = "@USE_FQDN_NRF@"; # Set to 'yes' if UPF relies on a DNS/FQDN service to resolve NRF's FQDN
- 
+       UPF_FQDN_5G  = "@UPF_FQDN_5G@";  #Set FQDN of UPF 
```

Comment:
@orion-belt @thinhnt1983 

This wlll require to modify at least a few docker-compose in the federation repository.

- `https://github.com/OPENAIRINTERFACE/openair-epc-fed/blob/master/docker-compose/magma-mme-demo/docker-compose.yml`
- `https://github.com/OPENAIRINTERFACE/openair-epc-fed/blob/master/docker-compose/oai-mme-legacy/docker-compose.yml`
- `https://github.com/OPENAIRINTERFACE/openair-epc-fed/blob/master/ci-scripts/dsTesterDockerCompose/docker-compose.tplt`
- `https://github.com/OPENAIRINTERFACE/openair-epc-fed/blob/master/ci-scripts/dsTesterDockerComposeLegacy/docker-compose.tplt`
- `https://github.com/OPENAIRINTERFACE/openair-epc-fed/blob/master/ci-scripts/oai-ran-sanity-check/spgwu.env`

Unless you put a default value in the `entrypoint.sh` like I did for the previous 5G fields.

Raphael